### PR TITLE
[stubs] add initial stubs for sat_clone

### PR DESCRIPTION
**Pull request by rplevka**

### CODE REVIEW

File: tests/foreman/sys/test_sat_clone.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        :CaseAutomation: notautomated
+
+        """
+
+    @destructive
+    @stubbed()
+    def test_clone_with_rename(self):
+        """Clone the satellite to other box
+
+        :id: 8dc01d48-5930-43a1-8a34-2fc150482541
```
Comment (rochacbruno) [inline tests/foreman/sys/test_sat_clone.py]: Duplicate `:id:` run `make uuid-fix`

File: tests/foreman/sys/test_sat_clone.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    @destructive
+    @stubbed()
+    def test_clone_without_rename(self):
+        """Clone the satellite to other box
+
+        :id: 8dc01d48-5930-43a1-8a34-2fc150482541
+
+        :Steps:
+            1. Refresh the manifest
+            Enable & sync new RH repo
```
Comment (rochacbruno) [inline tests/foreman/sys/test_sat_clone.py]: `rst` formatting error here, or you increase indentation or you add more listing numbers.

File: tests/foreman/sys/test_sat_clone.py (hunk 1 above)
Comment (rplevka) [inline tests/foreman/sys/test_sat_clone.py]: thanks

File: tests/foreman/sys/test_sat_clone.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+:Upstream: No
+
+:Setup:
+    1. Upload manifest
+    2. Enable and sync RH products
+    3. Create content views - publish, promote"
+    4. Create Activation keys
+    5. Create custom products, custom repos ((both yum and puppet), sync,
+    create content views - publish, promote
```
Comment (rochacbruno) [inline tests/foreman/sys/test_sat_clone.py]: Missing indentation here and causing travis to fail, run `make can-i-push` and also `make docs` locally before pushing.

Comment (rplevka) [inline tests/foreman/sys/test_sat_clone.py]: scream

**Comment by rplevka** [comment]
@rochacbruno comments addressed

**Pull request merged**